public/lib/helpers/timeline_helper.js

```diff
@@ -26,10 +59,15 @@ define(function (require) {
     TimelineHelper.prototype.pluckLabel = function (hit, params, notify) {
       let field;
       if (params.labelFieldSequence) { // in kibi, we have the path property of a field
-        field = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
```

Comment:
use logic from https://github.com/elastic/kibana/pull/7302/files#diff-b60fcd2a71f5585fbcbd967a546f5d53R168 to get the values from the `_source` first, and if missing, try to get them from `fields`.